### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by marimeireles**
Adding docs on how to make a pr

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -73,6 +73,27 @@ If you have questions about the project, using PyScript, or anything else, pleas
...
+
+After you fork PyScript make sure to add a new remote. You can do that using the command:
+
+```
+git remote add remote_nickname remote_url
+```
+
+You should have one remote that corresponds to your local repository and one remote that corresponds the upstream PyScript repository.
+
+We recommend you to create a new local branch and develop your changes in this brance=h. You can do that by using:
```
Comment (fpliger) [inline CONTRIBUTING.md:86]: typo brance=h -> branch

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -73,6 +73,27 @@ If you have questions about the project, using PyScript, or anything else, pleas
...
+
+```
+git checkout -b branch_name #creates new branch
+git checkout branch_name #goes to branch
+```
+
+The changes should try to be kept within the scope of the issue, or within a well defined scope that should be clearly explained on the PR title. If you can't think of a title that explains your PR well might be a sign that you need to split it in two or more PRs!
```
Comment (fpliger) [inline CONTRIBUTING.md:93]: Think this sentence could be rephrased to make it more readable. Something like "Please try to keep the changes on your PR within the scope of the issue,..."

Comment (fpliger) [reply CONTRIBUTING.md:93]: Also, you could add a `'` between "your PR" and "well"

**Review by fpliger — CHANGES_REQUESTED** [review]
minor changes but would be good to the these in :)

**Comment by marimeireles** [comment]
Thanks for the input @fpliger. Changed :)

**Review by fpliger — APPROVED** [review]
lgtm! TY @marimeireles !

**Pull request merged**